deployment.rb

```diff
@@ -218,7 +218,7 @@ def set_slogger_for_test(slogger)
   end
 
   def slog(params)
-    return unless slog_token
+    return unless slog_token && Gatekeeper.allows('activityMonitor', default: true)
```

Comment:
no strong preference but yeah, 'slogging' seems slightly more future-proof for a name here in case we happen to hook other things to `slog` later.